plugins/droid/droid-codec.cpp

```diff
@@ -92,9 +95,9 @@ class DroidVideoFrameMapper
         return true;
     }
 
-    gecko::camera::YCbCrFrame mapYcbCr(DroidMediaCodecData *decoded)
+    YCbCrFrame mapYcbCr(DroidMediaCodecData *decoded)
```

Comment:
This may be pre-existing, but is this capitalisation correct?
```suggestion
    YCbCrFrame mapYCbCr(DroidMediaCodecData *decoded)
```

Reply:
You're right. Fixed.